Source/radiation/Radiation.cpp

```diff
@@ -1085,42 +1085,79 @@ void Radiation::nonconservative_energy_update(Real& relative, Real& absolute,
 {
   BL_PROFILE("Radiation::nonconservative_energy_update");
 
-  relative = 0.0;
-  absolute = 0.0;
+  ReduceOps<ReduceOpMax, ReduceOpMax> reduce_op;
+  ReduceData<Real, Real> reduce_data(reduce_op);
+  using ReduceTuple = typename decltype(reduce_data)::Type;
+
+  const Real theta = 1.0;
+  const Real tiny = 1.e-50_rt;
 
 #ifdef _OPENMP
 #pragma omp parallel
 #endif
   {
-      Real relative_priv = 0.0;
-      Real absolute_priv = 0.0;
-      Real theta = 1.0;
       FArrayBox c_v;
 
       for (MFIter mfi(eta,true); mfi.isValid(); ++mfi) {
```

Comment:
Can you make the MFIter argument be `TilingIfNotGpu()`?

Reply:
done